docs/approvals/service-now/index.md

```diff
@@ -0,0 +1,151 @@
+![](images/servicenow-connections-2.png)
+
+Press **TEST** to ensure that the connection details are working. 
+
+Multiple ServiceNow connections are supported, however, each project can only use one ServiceNow connection.
+
+## Configuring deployments
+
+To enforce a deployment to require an approved CR, the **Change Controlled** setting needs to be enabled in **both** the project and the environment it is being deployed to.
+
+### Setting up projects for CR approval
+
+To enable a project to enforce a requirement for an approved CR, navigate to the project and then **{{Settings,General}}**.
+Check the **Change-controlled** setting and select your ServiceNow connection in the **Service Now Connection** setting, and then press **SAVE**.
+
+![](images/servicenow-project-settings.png)
+
+### Standard Change Templates
+By default, the CR creation will result in a `Normal` change. Setting the **Change Template Name** setting under **Project Settings** to the name of a valid, approved **Change Template** will result in a `Standard` change being created based upon the change template.
+
+### Supplying the CR number to a deployment
+
+If you add a variable to your project called `Octopus.ServiceNow.Change.Number`, then a CR will not be created and only the supplied CR number will be used to check for approval. This variable can also be [Scoped](/docs/projects/variables/index.md#scoping-variables) or configured as a [Prompted variable](/docs/projects/variables/prompted-variables.md).
+
+### Setting up environments for CR approval
+
+To enable an environment to enforce a requirement for an approved CR, navigate to **{{Infrastructure,Environments}}**, edit the environment via the overflow menu and check the **Change Controlled** setting, and then press **SAVE**.
+
+![](images/servicenow-environment-settings.png)
+
+## How it works
+
+Deployments where both the project and environment have **Change Controlled** enabled, will query ServiceNow for an approved CR before execution can begin.
+
+When a **Change Controlled** deployment is evaluated for approval, the following checks are performed:
+- If a specific CR number is available, via a variable named `Octopus.ServiceNow.Change.Number`, then only this CR will be checked.
+- If there is an existing CR with the specifically formatted **Short Description** available. See [Title text matching](#title-text-matching) for more information, then this CR will be evaluated.
+- Create a new CR. 
+  - This will be a `Normal` change, or a `Standard` change if the project has a `Change Template Name` set.
+  - A CR created by Octopus will have a **Short Description** in the format outlined in [Title text matching](#title-text-matching).
+
+Once a CR has been found, the deployment will only proceed if the **State** of the CR is `Implement`. If the **State** is either `New`, `Assess`, `Authorize`, or `Scheduled` the deployment will wait. Any other **State** will cause the deployment task to fail.
```

Comment:
This effectively says we're using the default change-lifecycle, but is it worth stating explicitly?

Reply:
yeah, that might be a good idea to call out.